.circleci/config.yml

```diff
@@ -36,6 +36,14 @@ jobs:
                 name: Install twine
                 command: |
                   pip3 install twine
+            - deploy:
```

Comment:
For the same of temporal complexity it's worth placing the new steps before twine as it's used when creating the package.